pkg/controller/storagecluster/reconcile.go

```diff
@@ -275,6 +290,55 @@ func (r *ReconcileStorageCluster) Reconcile(request reconcile.Request) (reconcil
 	return reconcile.Result{}, phaseErr
 }
 
+// getNodeTopologyMap
+func (r *ReconcileStorageCluster) getNodeTopologyMap(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
```

Comment:
If this function is expected to write topologyMap spec to server then this should probably be renamed to setNodeTopologyMap?

Why do we have to pass reqlogger? ReconcileStorageCluster already has that as a property if needed.

Reply:
You can't set what you don't have. This is very specifically getting information and then saving it somewhere.

As for reqLogger, I was just blindly following conventions elsewhere in the file. :) I can change that.